### DISCUSSION TITLE
Scenario issue grouping in allure report

### DISCUSSION THREAD

**Discussion by ymouhat** [allure-framework/allure2#2657] [Bug Reports]
Hi
using allure report the way the scenario are grouped in the report is misleading the user. 

As an example azure pipeline is executing the same test in different package.  
package1
-smoke-test-static.feature: this test contains the tag @package1 and the necessary tests for package1
package2
-smoke-test-static.feature: this test contains the tag @package2 and the necessary tests for package2
package3
-smoke-test-static.feature: this test contains the tag @package3 and the necessary tests for package3

Another reporting tool (cucumber-html-report) is able to split the test has it detects it is different feature files. and tags within the feature file.

![image](https://github.com/user-attachments/assets/24d051d7-f471-4fba-8673-0b0cbd48e3fb)

using allure report, all scenario are grouped under a single test

![image](https://github.com/user-attachments/assets/872953f1-1e71-4bf2-886c-f1e33318736b)

![image](https://github.com/user-attachments/assets/1a0e59b9-5dbd-44a0-a579-44e88e6662ff)

The only workaround I am thinking of is to change the name of the test 
(e.g.  smoke-test-static-package1.feature) 

However,is there any possibiltiy for allure to get as many tests as feature files and not to regroup them under a single test? 

Issue reproduced  with Aure devops Pipeline
azure extension:  Allure Report / Qameta Software / version 1.4.2
Allure-version 2.29
@wdio/cucumber-framework: 8.12.1

Thank you in advance,

**Comment by baev** [comment]
Could you please specify the Allure Cucumber integration package and the version?

**Reply by ymouhat** [reply]
Hi @baev
I have added the information .
Please let me know if you need something else?

**Reply by baev** [reply]
https://www.npmjs.com/package/@wdio/cucumber-framework ?

**Reply by ymouhat** [reply]
yes @wdio/cucumber-framework

**Reply by baev** [reply]
Thanks, we'll check it out.
